src/jsifier.js

```diff
@@ -58,6 +58,11 @@ function demangleCSymbolName(f) {
   return f[0] == '_' ? f.substr(1) : '$' + f;
 }
 
+function isJsLibraryConfigIdentifier(ident) {
```

Comment:
This looks like code from an unmerged PR? Or complementary to it.

As I mentioned in that PR, I think this can also be used in `library_webgl.js`.